### ISSUE TITLE
Julia v1.1 support

### ISSUE THREAD

**Issue by iblis17** [apache/incubator-mxnet#13954]
Julia v1.1 just release .... about 3hr ago.
https://github.com/JuliaLang/julia/releases/tag/v1.1.0

**Comment by frankfliu** [comment]
@mxnet-label-bot add [feature request]

**Comment by iblis17** [comment]
yeah, the PR #12845 has been merged. It will be published with 1.5.x release.

**Comment by eric-haibin-lin** [comment]
We should let the Julia community know this awesome work!

**Comment by lewisl** [comment]
Running 1.5.0 on Julia 1.1. Only did the tutorial mnist example so far--all worked well.  Will dig in more in the coming weeks.
